packages/wallet/src/services/TipTracker.ts

```diff
@@ -62,20 +66,34 @@ export class TipTracker extends PersistentDocumentTrackerSubject<Cardano.Tip> {
           connectionStatus$
         ]).pipe(
           // Throttle syncing by interval, cancel ongoing request on external trigger
+          tap(([isSettled, connectionStatus]) => {
+            if (connectionStatus === ConnectionStatus.down) {
+              // REVIEW: slightly redundant since connection debug messages are already recorded separately
+              logger.debug('Connection down. Skipping request');
+            } else {
+              logger.debug(isSettled === null ? 'Initial request' : 'Start request');
```

Comment:
```suggestion
              logger.debug('Fetching tip...');
```

Reply:
I'm wondering if we could measure the time between every `isSettled === true` (and from startup to the 1st one), without `delay(minPollInterval)` so we'd know how long it takes the wallet to make all requests. Would be a great log :thinking:

Reply:
Done in [bd1f8c0](https://github.com/input-output-hk/cardano-js-sdk/pull/379/commits/bd1f8c0a0a499a2060572ee911828d2848edbb9d)